meson.build

```diff
@@ -273,6 +273,10 @@ if libgcab.type_name() == 'pkgconfig'
   endif
 endif
 
+# manpages
+pandoc = find_program('pandoc', required: false)
```

Comment:
I think it should be something like `required: get_option('man')`, no?

Reply:
I think that only works if man is a boolean. Maybe that makes sense -- and then we do pandoc.found() around each generation block?

Reply:
Okay, how about this?